exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -34,9 +34,7 @@ import (
 	"github.com/thrasher-corp/gocryptotrader/log"
 )
 
-var (
-	errParsingWSField = errors.New("error parsing WS field")
-)
+var errParsingWSField = errors.New("error parsing WS field")
```

Comment:
```suggestion
var errParsingWSField = errors.New("error parsing WS field")
```

Up to you whether you want to do this in this PR, but you can delete this and use common.ErrParsingWSField instead. Looks like there's 4 areas where this is used in `bitfinex_websocket.go`

Reply:
Oh it's fine, already have some cleanup PR's of my own so will include it in that. Will merge this one as is